src/components/SaveTemplateButton/index.jsx

```diff
@@ -62,17 +63,29 @@ class SaveTemplateButton extends React.Component {
       setMode,
       saveTemplate,
       templateType,
+      emailTemplateSource,
+      emailTemplates,
```

Comment:
Why do we need to have this here ... we should use the state instead ... i.e. we can keep a state of number of emailTemplates.